src/feature/blogs/components/NewBlog.tsx

```
@@ -0,0 +1,261 @@
+                                                className='min-h-[500px]'
+                                                rows={20}
+                                                form={form}
+                                            />
+                                        </>
+                                    ) : (
+                                        <div className="prose prose-gray max-w-none">
+                                            <h1 className="text-3xl font-bold mb-4">{form.getValues('title') || "Untitled Post"}</h1>
+                                            <div className="text-base leading-relaxed">
+                                                {form.getValues('content') ? (
+                                                    <ReactMarkdown
+                                                        children={form.getValues('content')}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)